backtesting/_stats.py

```diff
@@ -137,6 +137,8 @@ def _round_timedelta(value, _period=_data_period(index)):
     s.loc['Profit Factor'] = returns[returns > 0].sum() / (abs(returns[returns < 0].sum()) or np.nan)  # noqa: E501
     s.loc['Expectancy [%]'] = returns.mean() * 100
     s.loc['SQN'] = np.sqrt(n_trades) * pl.mean() / (pl.std() or np.nan)
+    win_prob = (pl > 0).sum() / n_trades
+    s.loc['Kelly Criterion'] = win_prob - (1 - win_prob) / (pl[pl > 0].mean() / pl[pl < 0].mean())  # noqa: E501
```

Comment:
@t2o2 Since `pl[pl < 0].mean()` is negative, don't we need to prefix it with a minus sign? :thinking:

Reply:
ChatGPT is right :)

How do you check it using ChatGPT? Do you just paste the function into the conversation line?

I will push the PR shortly.

Reply:
I can't managed to get ChatGPT to detect this negative problem, would be interesting to know how you did it. Thanks.

Reply:
It took a few tries today after the recent update. But that's how I went about asking.
![image](https://user-images.githubusercontent.com/684364/208326451-e36f28cd-c671-466f-851f-2b6ded6ac89a.png)
This is just one lucky results. I think after their recent update, it's even more obvious the machine doesn't have a clue what it's doing. 😆